docs/apache-airflow-providers-databricks/operators/sql.rst

```diff
@@ -51,13 +51,21 @@ Operator executes given SQL queries against configured endpoint.  There are 3 wa
      - Optional parameters that will be used to substitute variable(s) in SQL query.
    * - session_configuration: dict[str,str]
      - optional dict specifying Spark configuration parameters that will be set for the session.
+   * - http_headers: list[tuple[str, str]]
```

Comment:
Guide shoudn:t list of all parameter. It should be available in reference. We should also not duplicate the documentation, as this makes it difficult to maintain.

Reply:
ok, I'll remove it - I simply followed the previous versions